play-frontend-hmrc-play-30/src/test/scala/uk/gov/hmrc/hmrcfrontend/config/TrackingConsentConfigSpec.scala

```diff
@@ -50,9 +50,21 @@ class TrackingConsentConfigSpec extends AnyWordSpec with Matchers {
       config.optimizelyUrl should equal(None)
     }
 
+    "return None if optimizely.projectId is null" in {
```

Comment:
sort of tested implicitly already, but wanted to add something explicit just to make sure I'm not accidentally breaking stuff 😇